### ISSUE TITLE
Suggest does nothing in some UD treebanks

### ISSUE THREAD

**Issue by stranak** [ufal/pmltq-suggest-server#10]
When I click Suggest, nothing happens.

Tested in Safari and Firefox, UD 2.5 Classical Chinese, and UD 2.4. Chinese PUD.

**Comment by matyaskopp** [comment]
this find:
https://github.com/ufal/pmltq-suggest-server/blob/master/script/pmltq-suggest-server#L36
does not follow symlinks so treex resources are missing if they are not embeded to data

```
 /opt/print-server/tred-extensions/treex -> /opt/treex/lib/Treex/Core/share/tred_extension/treex
```

```
Error occured while reading '/opt/pmltq-data/ud25/treex/zh-pud/test-001-001.treex.gz' using backend Treex::PML::Backend::PML:
Couldn't open PML schema file 'treex_schema.xml'
File does not exist: treex_schema.xml
```

**Comment by matyaskopp** [comment]
fixed 13942f467ece003999cdf59c05b4d5dc09a3d959
